src/store.js

```diff
@@ -85,6 +85,16 @@ export default new Vuex.Store({
         state.nodes.splice(index, 1);
       }
     },
+    removeMessageRef(state, message) {
+      state.nodes.filter(({ definition }) => (
+        definition.$type === 'bpmn:IntermediateCatchEvent' ||
+        definition.$type === 'bpmn:StartEvent'
+      )
+        && definition.eventDefinitions && definition.eventDefinitions.find(({ messageRef }) => messageRef === message))
```

Comment:
I think using `includes` instead of `find` here would be more "intent revealing", since we only care about checking that the message ref matches, we don't actually need to retrieve a reference to it.

Reply:
`includes` didn't work on the associated array. So I tried `some`